### PR TITLE
Adding xiaomin-tusimple's coordinates

### LINKED ISSUE #7079: Please add me
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by xiaomin-tusimple**
@githubteacher  What do you think?
This pull request closes #7079

### CODE REVIEW

**Comment by rmwegle** [comment]
@xiaomin-tusimple This looks great.

File: pins/xiaomin-tusimple.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (brianamarie) [inline pins/xiaomin-tusimple.json]: @xiaomin-tusimple Try deleting this empty line at the top.

**Comment by brianamarie** [comment]
@xiaomin-tusimple There are still a few errors in your file that are keeping the tests from passing. [Click "Help Me Troubleshoot" for some specific ideas](https://services.github.com/on-demand/intro-to-github/edit-files-on-github), or [watch this video to get some help](https://vimeo.com/201011288). I left a comment about one, but I think you also may need to rename the file so it's in a `_pins/` directory, not `pins`. 

If you need or want help, let me know! If this hasn't been resolved after a few days pass, I'll go ahead and make the changes for you so that it can be merged in. 😄

**Comment by alcoryaki** [comment]
seems good

**Comment by JezBassinder** [comment]
@xiaomin-tusimple looks great

**Comment by DCHarvard** [comment]
@xiaomin-tusimple 
This looks great! 🎉

**Comment by hectorsector** [comment]
@xiaomin-tusimple I went ahead and fixed the syntax issues in 5698fcb. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by tomorrow, I will go ahead and do it just to keep the repository clean!

**Pull request merged**